CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## 0.12.0
+
+**Release date:** 2021-06-10
+
+This prerelease comes with an update to the Kubernetes and controller-runtime
+dependencies to align them with the Kubernetes 1.21 release.
+
+<some note about go-yaml changes here>
```

Comment:
@stefanprodan: not sure what to put here, but I know there is something.

Reply:
copy the breaking change section from here: https://github.com/fluxcd/image-automation-controller/pull/182